src/blueprints/tv2/factories/tv2-graphic-action-factory.ts

```diff
@@ -0,0 +1,138 @@
+import { Tv2BlueprintConfiguration } from '../value-objects/tv2-blueprint-configuration'
+import { Action, PieceAction } from '../../../model/entities/action'
+import { PieceInterface } from '../../../model/entities/piece'
+import { PieceType } from '../../../model/enums/piece-type'
+import { Tv2SourceLayer } from '../value-objects/tv2-layers'
+import { TransitionType } from '../../../model/enums/transition-type'
+import { Tv2GraphicTimelineObjectFactory } from '../value-objects/factories/tv2-graphic-timeline-object-factory'
+import { PieceLifespan } from '../../../model/enums/piece-lifespan'
+import { PieceActionType } from '../../../model/enums/action-type'
+
+export class Tv2GraphicActionFactory {
+  constructor(
+    private readonly graphicTimelineObjectFactory: Tv2GraphicTimelineObjectFactory
+  ) { }
+
+  public createGraphicActions(blueprintConfiguration: Tv2BlueprintConfiguration): Action[] {
```

Comment:
I am pretty sure that it should be `Graphics` and not `Graphic`.

Reply:
Unless i am mistaken, then each action only creates/alters one Graphic element.
The method in question creates multiple actions.

Thus it creates multiple actions (plural), that each creates/alters one graphic element (singular).

Reply:
I'm in favour of `Graphic` instead of `Graphics` as André mentioned it's multiple actions being created that each does a single thing to a Graphic.

Reply:
Has been renamed to be with 's'.